### PR TITLE
Gh531 terms and privacy policy

### LINKED ISSUE #531: Introduction screens after install: Privacy Policy, T&C

### PR DESCRIPTION
**Pull request by bassemmagdy**
Please note that clearing localStorage will be helpful after merging this PR because there is a slight change made to the profile model. resolves #531

### CODE REVIEW

File: src/language/en-US.json [hunk 1]
```diff
@@ -165,7 +165,7 @@
     "download": "Download",
     "duration": "Duration",
     "duration_can_not_less_than_0": "Duration cannot be less than 0",
-    "duration_can_not_more_than_1_years": "Duration cannot be greater than 1 year",
+    "duration_can_not_more_than_1_years": "Duration cannot be greater 1 year",
```
Comment (yilmazbahadir) [inline src/language/en-US.json:168]: This line shouldn't be changed I believe.

File: src/language/en-US.json [hunk 2]
```diff
@@ -193,7 +193,7 @@
     "fee_speed_fast": "Fast",
     "fee_speed_fastest": "Fastest",
     "fee_speed_free": "No Fee",
-    "fee_speed_highest": "Highest",
+    "fee_speed_highest": "Fastest",
```
Comment (yilmazbahadir) [inline src/language/en-US.json:196]: Also this line.

File: src/language/en-US.json [hunk 3]
```diff
@@ -326,7 +326,6 @@
     "loading": "Loading...",
     "locked_mosaic": "locked mosaic",
     "login": "Login",
-    "low_fee_warning_message": "Transaction with a less than recommended fee might not get accepted and eventually",
```
Comment (yilmazbahadir) [inline src/language/en-US.json:329]: This line shouldn't be removed. I think you had some merge conflicts were not solved properly.

**Comment by rg911** [comment]
Added @AdriaCarrera as a reviewer

**Review by rg911 — APPROVED** [review]

**Review by Peersyst — APPROVED** [review]
Good job

**Pull request merged**